cursorless-talon/src/cursorless_snippets.talon

```diff
@@ -8,4 +8,4 @@ tag: user.cursorless
     user.cursorless_single_target_command(cursorless_insert_snippet_action, cursorless_positional_target, cursorless_insertion_snippet)
 
 {user.cursorless_insert_snippet_action} {user.cursorless_insertion_snippet_single_phrase} <user.text> [{user.cursorless_phrase_terminator}]:
-    user.cursorless_insert_snippet_with_phrase(cursorless_insert_snippet_action, cursorless_insertion_snippet_single_phrase, text)
+    user.private_cursorless_insert_snippet_with_phrase(cursorless_insert_snippet_action, cursorless_insertion_snippet_single_phrase, text)
```

Comment:
we should really do this for all of our private actions.  Figured I'd do it here tho to make it super clear that this is not one of the public snippet api functions

Reply:
I agree but I think it should be `cursorless_private`. Once we have actual Talon namespaces all of our actions will be prefixed with cursorless.

Reply:
I'm pretty sure we had this discussion a while back and decided on `private_cursorless`, so that it was super obvious that it's not part of our public api.  That way our public api is everything that starts with `cursorless_`.  Re using `_cursorless`, I'm pretty sure we're not allowed to prefix our actions with `_` in Talon